src/test/java/games/strategy/triplea/ui/RouteTest.java

```diff
@@ -31,15 +31,12 @@
   private final MapData dummyMapData = mock(MapData.class);
   private final MapRouteDrawer spyRouteDrawer = spy(new MapRouteDrawer(mock(MapPanel.class), dummyMapData));
   private final double[] dummyIndex = spyRouteDrawer.createParameterizedIndex(dummyPoints);
-  private final Route dummyRoute = spy(new Route());
+  private final Route dummyRoute = spy(new Route(mock(Territory.class), mock(Territory.class)));
```

Comment:
Use `Route(Territory, Territory...)` instead of `Route()` and `Route#add()` to ensure `m_start` is initialized correctly.